### PR TITLE
wallet: fix warning: "argument name 'feerate' in comment does not match parameter name"

### PR DESCRIPTION
**Pull request by furszy**
Should solve the tiny https://github.com/bitcoin/bitcoin/pull/25005#issuecomment-1159403854.

Which merely happens for the extra "=" character after the comma.

### CODE REVIEW

**Comment by Empact** [comment]
Thoughts on flipping this around, to properly document the args?:
```C++
   return AvailableCoins(wallet, coinControl,
        /*feerate=*/ std::nullopt, 
        /*nMinimumAmount=*/ 1,           
        /*nMaximumAmount=*/ MAX_MONEY,
        /*nMinimumSumAmount=*/ MAX_MONEY,
        /*nMaximumCount=*/ 0
    ).total_amount;
```

**Comment by furszy** [comment]
> Thoughts on flipping this around, to properly document the args?

Flipped 👍🏼

File: src/wallet/spend.cpp [hunk 1]
```diff
@@ -232,14 +232,13 @@ CoinsResult AvailableCoinsListUnspent(const CWallet& wallet, const CCoinControl*
...
-                          MAX_MONEY,    /*nMaximumAmount*/
-                          MAX_MONEY,    /*nMinimumSumAmount*/
-                          0             /*nMaximumCount*/
-                          ).total_amount;
+    return AvailableCoins(wallet, coinControl,
+            /*feerate=*/ std::nullopt,
+            /*nMinimumAmount=*/ 1,
+            /*nMaximumAmount=*/ MAX_MONEY,
+            /*nMinimumSumAmount=*/ MAX_MONEY,
+            /*nMaximumCount=*/ 0
```
Comment (Empact) [inline src/wallet/spend.cpp:240]: nit: could de-dent this to the standard 4 spaces

**Comment by Empact** [comment]
Code Review ACK https://github.com/bitcoin/bitcoin/pull/25410/commits/7ca8726f6355f8738e9c49533d028123051a624b

**Comment by w0xlt** [comment]
Post-merge ACK https://github.com/bitcoin/bitcoin/pull/25410/commits/7ca8726f6355f8738e9c49533d028123051a624b

**Pull request merged**